projects/igniteui-angular/src/lib/directives/text-highlight/text-highlight.directive.ts

```diff
@@ -328,14 +312,14 @@ export class IgxTextHighlightDirective implements AfterViewInit, AfterViewChecke
     }
 
     /**
-     * Activates the highlight if it is on the currently active row, column and page.
+     * Activates the highlight if it is on the currently active row and column.
      */
     public activateIfNecessary(): void {
         const group = IgxTextHighlightDirective.highlightGroupsMap.get(this.groupName);
-        const column = group.columnIndex === undefined ? group.column : group.columnIndex;
-        const row = group.rowIndex === undefined ? group.row : group.rowIndex;
+        const column = group.column === undefined ? group.column : group.column;
+        const row = group.row === undefined ? group.row : group.row;
```

Comment:
These conditions doesn't make sense after the rowIndex/columnIndex removal.

Reply:
Yeah, I guess remove the declarations and just use `group.column` and `group.row`

Reply:
Shame on me. I'll fix it :)